chaos_genius/views/rca_view.py

```diff
@@ -46,6 +49,40 @@ def kpi_get_line_data(kpi_id):
         message = str(err)
     return jsonify({"status": status, "message": message, "data": data})
 
+@blueprint.route("/<int:kpi>/download_chart_data", methods=["GET"])
+def kpi_download_line_data(kpi):
+    """API endpoint to download chart data for last 60 days."""
+    try:
+        line_data = kpi_line_data(kpi, download=True)
+        if line_data[0]=="error":
```

Comment:
(flake8) E225 missing whitespace around operator

```suggestion
        if line_data[0] == "error":
```